### PR TITLE
Gha accept test

### PR DESCRIPTION
**Pull request by jflo**

### CODE REVIEW

**Review by jflo — COMMENTED** [review]
LGTM

File: .github/workflows/acceptance-tests.yml [hunk 1]
```diff
@@ -43,22 +76,39 @@ jobs:
...
       - name: Upload Acceptance Test Results
+        if: steps.needsRun.outputs.result == 'true'
         uses: actions/upload-artifact@v3.1.0
         with:
           name: acceptance-node-${{matrix.runner_index}}-test-results
           path: 'acceptance-tests/tests/build/test-results/acceptanceTest/TEST-*.xml'
       - name: Publish Test Report
         uses: mikepenz/action-junit-report@v4
-        if: success() || failure() # always run even if the build step fails
+        if: steps.needsRun.outputs.result == 'true' && (success() || failure()) # always run even if the build step fails
```
Comment (jflo) [inline .github/workflows/acceptance-tests.yml:112]: I hope this syntax is right

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 jobs:
   dockerPromoteX64:
-    runs-on: [besu-research-ubuntu-8]
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-java@v3
         with:
-          distribution: 'temurin' # See 'Supported distributions' for available options
+          distribution: 'adopt' # See 'Supported distributions' for available options
```
Comment (fab-10) [inline .github/workflows/release.yml:13]: temurin is the right one, adopt is deprecated

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -86,21 +78,15 @@ jobs:
...
           architecture: arm64
         with:
-          arguments: testDocker
+          arguments: testDocker -PdockerOrgName=${{ secrets.DOCKER_ORG }}
       - name: Login to DockerHub
         uses: docker/login-action@v2
-        if: ${{ github.actor != 'dependabot[bot]' && (contains('refs/heads/release-', github.ref) || github.ref == 'refs/heads/main') }}
         with:
-          username: ${{ secrets.DOCKER_USER_RW }}
+          username: robocopsgonemad
```
Comment (fab-10) [inline .github/workflows/docker.yml:85]: remember to change this

**Review by fab-10 — APPROVED** [review]
test approval

**Review by jflo — COMMENTED** [review]
probably ready for main

**Comment by fab-10** [comment]
Before promoting to main, could you give an overview of what and how it has been done? 
Then I think we should wait after the holidays to merge this.

**Pull request merged**